### ISSUE TITLE
Cannot run on Ubuntu bionic

### ISSUE THREAD

**Issue by probonopd** [eckon/quick-clone#1]
Hello, thanks for making an AppImage. However, it was built on a too new system and hence cannot run on Ubuntu bionic:

![](https://github.com/AppImage/appimage.github.io/blob/fbd6dde06c46170fa2e69f7891594fa3286ae64a/database/Quick_Clone/screenshot.png?raw=true)

Please build on an older system, no newer than bionic. Thanks!

Reference:
https://github.com/AppImage/appimage.github.io/pull/2755

**Comment by eckon** [comment]
Hey, thanks for the reply.

I tried generating the image with Ubuntu 18.04 LTS, hope that fixes that.

Updated the release (https://github.com/eckon/quick-clone/releases/download/v0.1.0/quick_clone-v0.1.0-x86_64.AppImage) for https://github.com/AppImage/appimage.github.io/pull/2755

**Comment by probonopd** [comment]
Thanks @eckon. Re-running the test.

**Comment by probonopd** [comment]
Now it is complaining about a missing config file. Doesn't it create one automatically at first run?

![image](https://user-images.githubusercontent.com/2480569/134785811-adb87050-e22d-4b9a-99a1-abeffa6f1e8a.png)

**Comment by eckon** [comment]
Hey @probonopd, so far this is the intended behavior, but I might change it the next few days, creating one for ease of use.

**Comment by probonopd** [comment]
That would be great. I think adding it like this to AppImageHub is not nice because an error message as a screenshot doesn't give the awesome first impression your software certainly deserves ;-)

**Comment by eckon** [comment]
Thanks :)
Will update you in this issue when it's done

**Comment by eckon** [comment]
Hey @probonopd,
I added the small feature of creating a basic config file incase there is not already one.

Also generated the AppImage with github actions, but for some reasons now the image does not work on WSL anymore. Added for that case another image that I generated manually on a VM just in case.
Both should work on ubuntu either way.

It can be found in https://github.com/eckon/quick-clone/releases/tag/v0.2.0

Hope everything works :)
